action.yml

```diff
@@ -1,10 +1,23 @@
 name: 'Lighthouse CI Action'
 description: 'Run Lighthouse in CI using Github Actions'
 inputs:
-  url:
-    description: 'Page URL'
   urls:
-    description: 'List of URLs'
+    description: 'List of URL(s) to analyze'
+  static_dist_dir:
+    description: 'A build directory where your HTML files to run Lighthouse on are located.'
+  runs:
+    description: 'Number of runs to do per URL'
+    default: '3'
```

Comment:
I really think 3 is needed to do any assertions on, since I have seen some vm's startup and absolutely tank scores, but `1` isn't unreasonable to start with. If it's a big problem we can change the default.

Reply:
Maybe, we could show the option explicitly in docs, otherwise fallback on LHCI default?
For example:
```yml
urls: |
   https://example.com
   https://example.com/blog
runs: 1
```